locale/de/LC_MESSAGES/pronterface.po

```diff
@@ -5,62 +5,76 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Pronterface\n"
-"POT-Creation-Date: 2023-02-04 17:27+0100\n"
-"PO-Revision-Date: 2023-02-05 13:00+0100\n"
+"POT-Creation-Date: 2024-01-20 18:55+0100\n"
+"PO-Revision-Date: 2024-01-20 19:00+0100\n"
 "Last-Translator: DivingDuck\n"
 "Language-Team: <>\n"
 "Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: pygettext.py 1.5\n"
-"X-Generator: Poedit 3.2.2\n"
+"X-Generator: Poedit 3.4.2\n"
 
+# Besseren Begriff suchen
 #: printrun\excluder.py:26
-msgid ""
-"Part excluder: draw rectangles where print instructions should be ignored"
-msgstr ""
-"Druckteileausschluss: Zeichne Rechtecke um Teile, die ignoriert werden sollen"
+msgid "Print Excluder"
+msgstr "Druckausschluss"
 
-#: printrun\excluder.py:27 printrun\excluder.py:29
-msgid "Reset selection"
+#: printrun\excluder.py:30 printrun\excluder.py:32
+msgid "Reset Selection"
 msgstr "Auswahl aufheben"
 
-#: printrun\gcodeplater.py:77 printrun\gcodeplater.py:78
-msgid "GCODE files (*.gcode;*.GCODE;*.g)"
-msgstr "GCODE Dateien (*.gcode;*.GCODE;*.g)"
+#: printrun\excluder.py:39
+msgid "Draw rectangles where print instructions should be ignored."
+msgstr ""
+"Druckteileausschluss: Zeichne Rechtecke um Teile, die ignoriert werden "
+"sollen."
 
-#: printrun\gcodeplater.py:199 printrun\gcodeplater.py:236
+#: printrun\excluder.py:40
+msgid ""
+"\n"
+"Excluder always affects all layers, layer setting is disregarded."
+msgstr ""
+"\n"
+"Der Ausschließer wirkt sich immer auf alle Schichten aus, die "
+"Schichteinstellung wird ignoriert."
+
+#: printrun\gcodeplater.py:79
+msgid "GCODE files"
+msgstr "GCODE Dateien"
+
+#: printrun\gcodeplater.py:207 printrun\gcodeplater.py:248
 msgid "Exported merged G-Codes to %s"
 msgstr "Exportiere zusammengeführte G-Codes nach %s"
 
-#: printrun\gcview.py:108
+#: printrun\gcview.py:109
 msgid "Invalid layer for injection"
 msgstr "Ungültige Schicht zur Injektion"
 
-#: printrun\gcview.py:116
+#: printrun\gcview.py:117
 msgid "Invalid layer for edition"
 msgstr "Ungültige Schicht zur Bearbeitung"
 
-#: printrun\gcview.py:439
+#: printrun\gcview.py:443
 msgid "Fit to plate"
 msgstr "Auf Druckplatte platzieren"
 
-#: printrun\gcview.py:440
+#: printrun\gcview.py:444
 msgid "Fit to plate [F]"
 msgstr "Auf Druckplatte platzieren [F]"
```

Comment:
This command zooms camera to the gcode-object until it fits the view of the 3d panel. Maybe "An Ansicht anpassen" describes this better.
Tbh now that I see it, I think "Fit to plate" should be "Fit to view" or something similar

Reply:
Good catch as well! You are correct, I just check our readme.md:

[3D VIEWER CONTROLS](https://github.com/kliment/Printrun#3d-viewer-controls): `F key: fit view to display entire print`

I will change the string in source code and update all templates and translation files.